packages/run-protocol/src/my-lien.js

```diff
@@ -30,9 +30,32 @@ export const makeStakeReporter = (bridgeManager, stake, denom = 'ubld') => {
   const { make: makeAmt } = AmountMath;
   /** @param {string} numeral */
   const toStake = numeral => makeAmt(stake, BigInt(numeral));
+  /**
+   * @param {string} address
+   * @param {bigint} delta
+   * @returns {Promise<Amount<`nat`>>}
+   */
+  const changeLiened = async (address, delta) => {
+    assert.typeof(address, 'string');
+    const newAmount = await E(bridgeManager).toBridge(XLien.name, {
+      type: XLien.LIEN_CHANGE_LIENED,
+      address,
+      denom,
+      delta: `${delta}`,
+    });
+    return harden(toStake(newAmount));
+  };
 
   /** @type {StakingAuthority} */
   const stakeReporter = Far('stakeReporter', {
+    increaseLiened: async (address, increase) => {
+      const delta = AmountMath.getValue(stake, increase);
```

Comment:
precedes this PR but `stake` reads like an amount, whereas it's the brand that is to be staked.

I think it's the only brand in scope so imo a (non-blocking) improvement would be to rename it `brand`.

Reply:
Done.